app/_src/gateway/install/linux/ubuntu.md

```diff
@@ -7,12 +7,62 @@ The {{site.base_gateway}} software is governed by the
 Kong is licensed under an
 [Apache 2.0 license](https://github.com/Kong/kong/blob/master/LICENSE).
 
+{:.note}
+> This page will install {{ site.base_gateway }} in traditional mode, where it acts as both the Control Plane and Data Plane. Running in this mode may have a small performance impact.
+> &nbsp;
+> 
+> &nbsp;
+> 
+> We recommend using [Konnect](https://konghq.com/products/kong-konnect/register?utm_medium=referral&utm_source=docs&utm_campaign=install-ubuntu) as your Control Plane to allow your Data Plane to run at maximum performance and decrease your deployment complexity
+
 ## Prerequisites
 
 * A supported system with root or [root-equivalent](/gateway/{{page.kong_version}}/production/running-kong/kong-user/) access.
 * (Enterprise only) A `license.json` file from Kong
 
-## Download and install
+## Installation
+
+The quickest way to get started with {{ site.base_gateway }} is using our install script:
+
+{% navtabs_ee codeblock %}
+{% navtab Kong Gateway %}
+```bash
+bash <(curl -sS https://install-script--getkong.netlify.app/install) -v {{page.versions.ee}}
+```
+{% endnavtab %}
+{% navtab Kong Gateway (OSS) %}
+```bash
+bash <(curl -sS https://install-script--getkong.netlify.app/install) -p kong -v {{page.versions.ce}}
+```
+{% endnavtab %}
+{% endnavtabs_ee %}
+
+This script will detect your operating system and automatically install the correct package. It'll also install a Postgres database and bootstrap {{ site.base_gateway }} for you.
+
+If you'd prefer to install just the package, please see the [Package Install](#package-install) section.
```

Comment:
I think it's just my lack of understanding, but why would you want to just install the package? Or maybe conversely, why would you want to install Gateway, Postgres, and bootstrap Gateway at the same time with the script? But maybe the target audience would already know the answer to this and wouldn't need it spelled out.

Reply:
This comes down to the target audience for me. Either:

* They know why they need a package only and have the skills required to manage a package install + external DB
* They don't know why they need a package only install, and the install script is what they should use

If we get more questions on this we can expand the _why_ but for now I'd like to keep the page streamlined